admin/client/ChartEditor.ts

```diff
@@ -10,7 +10,7 @@ import { ChartConfig } from "charts/ChartConfig"
 import { EditorFeatures } from "./EditorFeatures"
 import { Admin } from "./Admin"
 import { BAKED_GRAPHER_URL } from "settings"
-import _ = require("lodash")
+import _ from "lodash"
```

Comment:
nit: anyone else in favor of doing "lodash" instead of "_"? Similarly "jQuery" instead of "$". With autocomplete nowadays I find it slightly better to use the full name (I find it a little more informative to do ad hoc back-of-the-envelope queries of the code base to answer questions like "how much are we using lodash").

Reply:
Would be all for that, but maybe we should do that in another PR (that's less risky than this one) so it is easier to review everything that's going on here? The other one should be a quick one then (and hopefully we can one day get rid of jQuery anyway!)

Reply:
Yes agreed it's best for another PR.